openedx/features/course_experience/views/course_outline.py

```diff
@@ -140,13 +140,16 @@ def user_enrolled_after_completion_collection(self, user, course_key):
         """
         begin_collection_date = datetime.datetime(2018, 01, 24, tzinfo=pytz.utc)
         user = User.objects.get(username=user)
-        user_enrollment = CourseEnrollment.objects.get(
-            user=user,
-            course_id=course_key,
-            is_active=True
-        )
-
-        if user_enrollment and user_enrollment.created > begin_collection_date:
+        try:
+            user_enrollment = CourseEnrollment.objects.get(
+                user=user,
+                course_id=course_key,
+                is_active=True
+            )
+        except CourseEnrollment.DoesNotExist:
+            return False
+
+        if user_enrollment.created > begin_collection_date:
```

Comment:
Why not just wrap this statement into the `try:` clause? I think logically it belongs there better.

Reply:
Oh, yeah that makes sense.